bigbluebutton-tests/playwright/layouts/focusOnVideo.js

```diff
@@ -0,0 +1,34 @@
+const { expect } = require('@playwright/test');
+const Page = require('../core/page');
+const e = require('../core/elements');
+const { ELEMENT_WAIT_LONGER_TIME } = require('../core/constants');
+const { MultiUsers } = require('../user/multiusers');
```

Comment:
imports not being used
```suggestion
const { expect } = require('@playwright/test');
const e = require('../core/elements');
const { MultiUsers } = require('../user/multiusers');
```